crates/nodeapi/src/lib.rs

```diff
@@ -146,18 +146,32 @@ impl NodeApi {
 }
 
 #[allow(unused)]
-pub async fn node_info(api: web::types::State<NodeApi>) -> impl Responder {
+pub async fn node_info(api: web::types::State<Arc<Mutex<NodeApi>>>) -> impl Responder {
```

Comment:
This should be private. Also, let's remove the `#[allow(unused)]`

Reply:
done